### PR TITLE
[stable/prometheus-adapter] Upgrading to v0.6.0

### PR DESCRIPTION
**Pull request by mentos1386**
#### What this PR does / why we need it:
A new version was released yesterday https://github.com/DirectXMan12/k8s-prometheus-adapter/releases/tag/v0.6.0

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

@mattiasgees @steven-sheehy @hectorj2f

### CODE REVIEW

File: stable/prometheus-adapter/Chart.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: prometheus-adapter
-version: 2.0.1
-appVersion: v0.5.0
+version: 2.0.2
```
Comment (hectorj2f) [inline stable/prometheus-adapter/Chart.yaml]: this should be a 2.1.0 version

Comment (mentos1386) [reply stable/prometheus-adapter/Chart.yaml]: fixed

**Comment by mentos1386** [comment]
@hectorj2f can you give it another look? I have fixed the `version` issue.

**Pull request merged**